api_factory/schema/wrappers.py

```diff
@@ -187,4 +193,22 @@ def pb2_modules(self) -> Sequence[Tuple[str, str]]:
     @property
     def has_lro(self) -> bool:
         """Return whether the service has a long-running method."""
-        return any(method.lro_payload for method in self.methods.values())
+        return self._any_method('lro_payload')
+
+    @property
+    def has_field_headers(self) -> bool:
+        """Return whether the service has a method containing field headers."""
+        return self._any_method('field_headers')
+
+    def _any_method(self, attr: str) -> bool:
```

Comment:
Clever. I wonder if this is a concept worth trying to expose more generically.

What if we sent down `any` and `all` methods that accepted an iterable for the first argument and lambda function for the second?

Not sure if Jinja syntax would support sending the lambda though.

Reply:
I don't think this is possible in Jinja due to allowing only "view logic." I could make the function take in a lambda from its callers within the same wrapper class instead.